Sources/Sentry/SentrySessionReplayIntegration.m

```diff
@@ -459,7 +459,7 @@ - (void)sentrySessionEnded:(SentrySession *)session
     _sessionReplay = nil;
 }
 
-- (void)sentrySessionStarted:(SentrySession *)session
```

Comment:
Removed the parameter to avoid exposing it since it was not really used in the implementation